src/util/Injected.js

```diff
@@ -317,6 +317,21 @@ exports.LoadUtils = () => {
         return true;
     };
 
+    window.WWebJS.getLabels = async () =>{
+        return window.Store.Label.models.map( function (label){  return  { 'name': label.name , 'hexColor': label.hexColor , 'id':label.id }; });
+    };
+
+    window.WWebJS.getChatLabels = async (chatId) => {
+        let chatLabelsIndex = await window.WWebJS.getChat(chatId);
+        chatLabelsIndex = chatLabelsIndex.labels;
+        //console.log(chatLabelsIndex)
+        let res = [];
+        for (var i in chatLabelsIndex){
+            res.push({ 'name': window.Store.Label.get(chatLabelsIndex[i]).name , 'hexColor': window.Store.Label.get(chatLabelsIndex[i]).hexColor ,  'id' : chatLabelsIndex[i] } );    
```

Comment:
I'm seeing in multiple places you're constructing this object representation of a Label with the same properties. To keep things consistent and remove code duplication, perhaps we could instead have a serializer function, much like we have with `getChatModel` and `getMessageModel` that takes care of returning a properly serialized representation of the label with the necessary properties.